src/components/icons/IconStatus/IconStatus.tsx

```diff
@@ -0,0 +1,153 @@
+import * as React from 'react';
+
+import {
+  ExclamationCircleIcon,
+  InProgressIcon,
+  OffIcon,
+  PausedIcon,
+  SyncAltIcon,
+  UnknownIcon,
+} from '@patternfly/react-icons';
+import { global_danger_color_100 as dangerColor } from '@patternfly/react-tokens/dist/js/global_danger_color_100';
+
+interface IconProps {
```

Comment:
consider change to type :)

Reply:
what about data-test ? :)

Reply:
What do you mean with data-test? Should we add a prop to provide data-test attribute?

Reply:
Good idea!